datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -26,6 +26,7 @@
 # Integrations released for the last time by a revoked developper but not shipped anymore.
 EXCLUDED_INTEGRATIONS = [
     "datadog-docker-daemon",
+    "datadog-dd-cluster-agent",  # excluding this since actual integration is called `datadog-cluster-agent`
```

Comment:
Note to reviewer: the pipeline was failing for `datadog-dd-cluster-agent` since integration only had a beta release by a revoked developer. The actual integration is `datadog-cluster-agent`.